prov/util/src/util_attr.c

```diff
@@ -1078,8 +1078,8 @@ static void fi_alter_domain_attr(struct fi_domain_attr *attr,
 }
 
 static void fi_alter_ep_attr(struct fi_ep_attr *attr,
-			     const struct fi_ep_attr *hints,
-			     uint64_t info_caps)
+			     struct fi_ep_attr *hints,
```

Comment:
hints must remain const.  We should not be altering the user's provided hints.  We are only setting 'attr' in this function.